src/util/notifications.ts

```diff
@@ -11,10 +10,6 @@ export interface NotifOpts extends Notifications.CreateNotificationOptions {
 const onClickListeners = new Map<string, (id: string) => void>()
 
 browser.notifications.onClicked.addListener(id => {
-    internalAnalytics.processEvent({
```

Comment:
Was this meant to be removed?

Reply:
Yeah, we are sending the analytics when notification called, we also need notification id, so better to call the function there. :slightly_smiling_face: